common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -255,6 +255,59 @@ color_lists = {
 			125 = "black"
 			250 = "black_empire_dark_red"
 		}
+		
+		###### Warcraft Culture Specific CoAs ######
+		
+		#### Humans ###
+		
+		special_selection = { 
+			trigger = { azerothian_coa_trigger = yes }
+			125 = "gold_alliance"
```

Comment:
I believe it should go in `metal_colors`. These are the colors of the emblem.

`normal_colors` are the colors of the background.